Dockerfile

```diff
@@ -2,7 +2,6 @@ FROM gcr.io/kaggle-images/python-tensorflow-whl:1.12.0-py36 as tensorflow_whl
 FROM continuumio/anaconda3:5.2.0
 
 ADD clean-layer.sh  /tmp/clean-layer.sh
-ADD patches/ /tmp/patches/
```

Comment:
I removed this since it was causing a lot of layers to be rebuilt if patches was ever changed, and I couldn't see any use of /tmp/patches - but please let me know if there's something I'm overlooking!

Reply:
sgtm